frontend/cypress/integration/featureFiles/overview.feature

```diff
@@ -137,10 +137,64 @@ Feature: Kiali Overview page
     Then user sees the memory chart
     And user sees the cpu chart
 
+  @overview-page
+  @multi-cluster
+  @skip
+  Scenario: The badge for local cluster should be visible
```

Comment:
This is just a simple badge check. After seeing this PR (https://github.com/kiali/kiali/pull/6019), I think the local cluster name could be obtained somewhere from the configs, but I am not sure where to look for it exactly.